src/main/java/cl/transbank/webpay/Options.java

```diff
@@ -1,10 +1,13 @@
 package cl.transbank.webpay;
 
+import cl.transbank.common.Option;
 import lombok.*;
 import lombok.experimental.Accessors;
 
 @ToString @NoArgsConstructor @AllArgsConstructor
-public class Options implements Cloneable {
+public class Options extends Option implements Cloneable {
+    @Getter  final String headerCommerceCodeName = "Tbk-Api-Key-Id";
+    @Getter  final String headerApiKeyName = "Tbk-Api-Key-Secret";
     @Setter @Getter @Accessors(chain = true) private String commerceCode;
```

Comment:
`commerce_code` and `api_key` could be inherited from `Options` so you could delete them from here.

Reply:
got ya